dist.ini

```diff
@@ -1,8 +1,8 @@
--name    = Date-Holidays-GB
--author  = Michael Jemmeson <mjemmeson@cpan.org>
--license = Perl_5
--copyright_holder = Michael Jemmeson
--copyright_year   = 2020
+name    = Date-Holidays-GB
+author  = Michael Jemmeson <mjemmeson@cpan.org>
+license = Perl_5
```

Comment:
Without at least this line uncommented (I assume), I get this error when building:
https://github.com/peternewman/Date-Holidays-GB/runs/1746431574?check_suite_focus=true#step:8:1198

Reply:
It's because I earlier removed LicenseFromModule from the Milla bundle below, but I have no idea why? It seems to work now testing locally

Reply:
Do you mean it works without LicenseFromModule or it works unmodified (on a clean checkout)? It sort of seemed like Milla either needed to be shipped with, or generate a set of possible licences to match against and it wasn't doing that or was silently failing to do so.

Reply:
If the - is a comment/skip (which was my guess), then you didn't remove LicenseFromModule but you stopped removing it (hence needing to manually specify the licence).

I see you've fixed it in another way anyway.

Reply:
the `-` is to pass the option to the filter bundle, so it's incorrect on those fields (name, etc) and I'm not sure why it was there! (`;` is comment in the `.ini` files)

Reply:
Ah, that's odd.